template.yaml

```diff
@@ -354,7 +355,69 @@ Resources:
       VersioningConfiguration:
         Status: Enabled
 
-  # Query Bucket
+  # Data Analysis
+
+  InitiateAthenaQueryFunction:
+    #checkov:skip=CKV_AWS_116:Lambda is behind an API gateway, so repeated processing of the same message isn't a problem
```

Comment:
In this case, I think you do want a dead-letter queue, because you're processing stuff off a queue. Have a look at what's now in the main branch, for ProcessDataRequestFunction

Reply:
Thanks, have removed the skip and have added a dead letter queue